service/messagecontext/message_context.go

```diff
@@ -0,0 +1,45 @@
+// Package messagecontext stores and accesses the message struct in
+// context.Context.
+package messagecontext
+
+import (
+	"context"
+)
+
+// key is an unexported type for keys defined in this package. This prevents
+// collisions with keys defined in other packages.
+type key string
+
+// messageKey is the key for message struct values in context.Context. Clients
+// use messagecontext.NewContext and messagecontext.FromContext instead of using
+// this key directly.
+var messageKey key = "message"
+
+// Message is a communication structure used to transport information from one
+// resource to another. Messages move between resource during reconciliation
+// within the dispatched context.
+type Message struct {
+	// ConfigMapNames is a list of config map names filled by the config map
+	// resource and read by the deployment resource.
+	ConfigMapNames []string
```

Comment:
What is the message context used for? I do not see it being used and the config map names here should be irrelevant as well.

Reply:
This was in the migration branch and I just copied it across. My bad, it's now removed.